### PR TITLE
Fixed a typo and implemented #148

### PR DESCRIPTION
**Pull request by henon**
`value` was repeatedly spelled `vlaue`

### CODE REVIEW

**Comment by henon** [comment]
By the way, I don't know why the doc generator generated different white space on my machine. 

I think there are two ways to address that: 
1) configure git to ignore whitespace
2) change the doc generator

what do you think?

File: src/MatBlazor/Components/MatSnackbar/BaseMatSnackbar.cs [hunk 1]
```diff
@@ -29,14 +32,41 @@ public bool IsOpen
                 if (IsOpen != value)
                 {
                     _isOpen = value;
-                    CallAfterRender(async () =>
+                    CallAfterRender(async () => await SetIsOpen(value));
+                    if (_isOpen == false)
```
Comment (lindespang) [inline src/MatBlazor/Components/MatSnackbar/BaseMatSnackbar.cs:36]: @henon I think this is a code smell unless it's a nullable bool (which it isn't in this case)

**Comment by lindespang** [comment]
@henon Thanks for this, I like the shift from JS to .NET - aim for MatBlazor is to do as little JS as possible.

I changed the base to develop (this should be mentioned in a PR template which I should write).

Also, do you mind cleaning this up a little? So that only the source is comitted (Not generated files). As you say there seems to be some issue with line conversion when you use the tool but it isn't really required since it's a build step before we publish it.

**Comment by henon** [comment]
I cleaned everything up and will start a new PR with my branch henon_Snackbar_timeout which is based on the develop branch.

**Pull request closed without merging**